static/app/components/hotkeys.tsx

```diff
@@ -0,0 +1,121 @@
+import styled from '@emotion/styled';
+
+import space from 'sentry/styles/space';
+import {getKeyCode} from 'sentry/utils/getKeyCode';
+
+const macModifiers = {
+  18: '⌥',
+  17: '⌃',
+  91: '⌘',
+};
+
+const normalModifiers = {
+  18: 'ALT',
+  17: 'CTRL',
+  91: 'CMD',
+};
+
+const genericGlyphs = {
+  16: '⇧',
+  8: '⌫',
+  37: '←',
+  38: '↑',
+  39: '→',
+  40: '↓',
+  9999: '+',
+};
+
+const keyToDisplay = (
+  key: string,
+  platform?: 'macos' | string
+): [React.ReactNode, boolean] => {
+  let keyStr = key.toUpperCase();
+  let hasMissing = false;
+
+  const keyCode = getKeyCode(key);
+  if (keyCode) {
+    const isMac = platform
+      ? platform === 'macos'
+      : window?.navigator?.platform?.toLowerCase().startsWith('mac') ?? false;
+
+    if (isMac) {
+      const modifier = macModifiers[keyCode];
+      if (modifier) {
+        keyStr = modifier;
+      }
+    } else {
+      const modifier = normalModifiers[keyCode];
+      if (modifier) {
+        if (modifier === 'CMD') {
+          hasMissing = true;
+        }
+        keyStr = modifier;
+      }
+    }
+
+    const glyph = genericGlyphs[keyCode];
+
+    if (glyph) {
+      keyStr = glyph;
+    }
+  }
+
+  // eslint-disable-next-line react/jsx-key
+  return [<Key>{keyStr}</Key>, hasMissing];
```

Comment:
Let's use an object instead of a tuple as a return, this eslint ignore is kinda gross too haha